### PR TITLE
Feature/itemizer

### PR DESCRIPTION
**Pull request by akashpatl**
Re-add all endpoints connecting  hedgehog to item-pricing service

### CODE REVIEW

File: src/main/java/com/hedvig/backoffice/services/itemPricing/dto/InventoryItemDTO.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
+package com.hedvig.backoffice.services.itemPricing.dto;
+
+import lombok.Value;
+import java.math.BigDecimal;
+import com.hedvig.backoffice.services.itemPricing.dto.FilterDTO;
+import java.util.List;
+
+@Value
+public class InventoryItemDTO {
```
Comment (vonElfvin) [inline src/main/java/com/hedvig/backoffice/services/itemPricing/dto/InventoryItemDTO.java]: ClaimInventoryItem

File: src/main/java/com/hedvig/backoffice/services/itemPricing/dto/PayloadDTO.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
+package com.hedvig.backoffice.services.itemPricing.dto;
+
+import lombok.Value;
+import java.util.List;
+import com.hedvig.backoffice.services.itemPricing.dto.FilterDTO;
+
+@Value
+public class PayloadDTO {
```
Comment (vonElfvin) [inline src/main/java/com/hedvig/backoffice/services/itemPricing/dto/PayloadDTO.java]: rename for calirty

File: src/main/java/com/hedvig/backoffice/services/itemPricing/dto/PricepointDTO.java [hunk 3]
```diff
@@ -0,0 +1,15 @@
+package com.hedvig.backoffice.services.itemPricing.dto;
+
+import lombok.Value;
+import java.math.BigDecimal;
+
+@Value
+public class PricepointDTO {
+  String _id;
```
Comment (vonElfvin) [inline src/main/java/com/hedvig/backoffice/services/itemPricing/dto/PricepointDTO.java]: _id?

File: src/main/resources/schema.graphqls [hunk 4]
```diff
@@ -8,11 +8,17 @@ type QueryType {
...
   paymentSchedule(status: ChargeStatus!): [SchedulerState]
+  categories: [Category!]
+  items(payload: Payload!): ItemSearch!
+  prices(date: String!, ids: [String!]!): [PricePoint!]!
   ticket (id: ID!): Ticket
   getFullTicketHistory( id: ID!): TicketHistory
   tickets (resolved: Boolean): [Ticket!]!
   getAnswerSuggestion(question: String): [Suggestion!]!
   me: String
+  inventory(claimId: ID!): [InventoryItem]
```
Comment (palmenhq) [inline src/main/resources/schema.graphqls]: is this array nullable? Shouldn't it just return an empty array if no items exist?

**Pull request merged**